site/docs/components/date-picker/examples.mdx

```diff
@@ -36,6 +36,14 @@ Date picker can show validation states (warning, success, error) with the prop `
 The Date Range Picker enables selection of start and end dates, using the input fields or two calendars displayed.
 
 </LivePreview>
+
+<LivePreview componentName="date-picker" exampleName="CompactRange">
+## Compact Date range
```

Comment:
I don't think we should keep this naming, Visible months is probably better

Reply:
modified